docker/code-check.Dockerfile

```diff
@@ -0,0 +1,7 @@
+# Image for all tasks related to static code analysis in Async-Profiler
+FROM public.ecr.aws/docker/library/amazoncorretto:11-alpine-jdk
+
+RUN apk add --no-cache clang-extra-tools linux-headers make wget python3 git && \
+    wget https://raw.githubusercontent.com/llvm/llvm-project/main/clang-tools-extra/clang-tidy/tool/clang-tidy-diff.py -O /usr/bin/clang-tidy-diff && \
```

Comment:
You can use `ADD` command to add a file to an image directly without need to install wget.

If the script is called `clang-tidy-diff.py`, why not leaving the original name?

Finally, use specific commit when referring to third-party tools on github.

Reply:
Thanks, didn't think about it: 75f6beb835c3fbfe2bfac29ba5e06b27f8e3fc44